packages/toast/toast-container.js

```diff
@@ -125,7 +127,7 @@ export class WarpToastContainer extends WarpElement {
   render() {
     return html`
       <aside class="${ccToaster.container}">
-        <div class="${ccToaster.toaster}" id="w-toast-container-list">
+        <div class="${ccToaster.base}" id="w-toast-container-list">
```

Comment:
I guess to make it consistent with the component name, this should be called `ccToastContainer` (and `toastContainer` in the css repo) but it's not a big deal :)

Reply:
ah good catch! I'll change it :)

Reply:
Updated in this commit: [bbcc86d](https://github.com/warp-ds/elements/pull/174/commits/bbcc86da0f56532e0971b2329382b191740a1cd6)